src/main/java/frc/robot/Constants.java

```diff
@@ -2,9 +2,43 @@
 
 
 public final class Constants {
+    public static final double NOMINAL_VOLTAGE = 12; // [volts]
 
     public static class ExampleSubsystem {
         public static final double POWER = 0.5; // [%]
     }
 
+    public static class Climber {
+        public static final double P_VELOCITY = 1.5;
+        public static final double I_VELOCITY = 0;
+        public static final double D_VELOCITY = 0;
+
+        public static final double F_FORWARD_S = 0;
+        public static final double F_FORWARD_COS = 0;
+        public static final double F_FORWARD_V = 0;
+        public static final double F_FORWARD_A = 0;
+
+        public static final double CRUISE_VELOCITY = 0; // [ticks/100ms]
+        public static final double ACCELERATION = 0; // [ticks/100ms*sec]
```

Comment:
Rename to `MAXIMAL_ACCELERATION` or `TARGET_ACCELERATION` (that's the wording used in CTRE's docs).